zerocode-maven-archetype/README.txt

```diff
@@ -0,0 +1,22 @@
+To install archetype locally:
+    Navigate to zerocode-maven-archetype on your machine. Run "mvn install" from this directory
+    
+To generate anew archetype-based project:
```

Comment:
Hi, Great job and thanks for the PR. 

I got some minor issues if you could fix it(when you get chance please). 🙏 

Just checking if I am doing it correctly in the following scenarios:
## 1)
```java
$ mvn archetype:generate -DarchetypeGroupId=com.myproject -DarchetypeArtifactId=zerocodeArchetype -DarchetypeVersion=1.0-SNAPSHOT -DgroupId=com.myproject -DartifactId=zerocodeArchetypeTest
```
works fine.

## 2)
```java
$ mvn archetype:generate -DarchetypeGroupId=com.myproject -DarchetypeArtifactId=zerocodeArchetype -DarchetypeVersion=1.0-SNAPSHOT -DgroupId=com.ms.xbox -DartifactId=ms-xbox
```
works fine.
But the package names as generated as `com.myproject` instead of `com.ms.xbox`

![image](https://user-images.githubusercontent.com/12598420/67792109-8075dd80-fa70-11e9-9418-6af96a68cbe5.png)

All lookin good otherwise!